src/models/pieChart.js

```diff
@@ -193,13 +184,23 @@ nv.models.pieChart = function() {
     // Event Handling/Dispatching (out of chart's scope)
     //------------------------------------------------------------
 
-    pie.dispatch.on('elementMouseover.tooltip', function(e) {
-        e.pos = [e.pos[0] +  margin.left, e.pos[1] + margin.top];
-        dispatch.tooltipShow(e);
+    pie.dispatch.on('elementMouseover.tooltip', function(evt) {
+        tooltip.html(tooltipContent(evt))
+            .style('display', 'inline-block');
+    });
+
+    pie.dispatch.on('elementMouseout.tooltip', function(evt) {
+        tooltip.style('display', 'none');
+    });
+
+    pie.dispatch.on('elementMousemove.tooltip', function(evt) {
+        tooltip.style('top', (event.pageY-15)+"px").style("left",(event.pageX+10)+"px");
```

Comment:
I considered that but I intentionally didn't since I'd like to get into some of this other tooltip code.  Ideally I'd like all the charts to use a tooltip object that has it's own options like the legend.